core/build.gradle

```diff
@@ -130,7 +130,8 @@ task (compileCssDebug, type: Exec) {
 
 task (copyNginx, type: Exec) {
   if (mac()) {
-    def brewLocation = 'which brew'.execute().text[0..-10]
+//  def brewLocation = 'which brew'.execute().text[0..-10] // use nginx from current machine
```

Comment:
We can probably kill this comment, huh?  Or are we going to go back to it in the near future?  Or does it add value to the new code?  If it's just to keep a record, we could always come back in source control if we need it, right?